cmd/clusterctl/client/cluster/installer.go

```diff
@@ -82,6 +95,12 @@ func installComponentsAndUpdateInventory(components repository.Components, provi
 	inventoryObject := components.InventoryObject()
 
 	log.V(1).Info("Creating objects", "Provider", components.ManifestLabel(), "Version", components.Version(), "TargetNamespace", components.TargetNamespace())
+	// log.Info("Installing the follow objects:")
+	// for _, obj := range components.Objs() {
+	// 	if obj.GroupVersionKind().Kind == "Deployment" {
+	// 		log.Info(fmt.Sprintf("-------\nname: %v\ngvk: %+v\nnamespace: %v\n", obj.GetName(), obj.GroupVersionKind(), obj.GetNamespace()))
+	// 	}
+	// }
```

Comment:
Left over?

Reply:
This is still a WIP PR. 
I will do a full clean up later. :)